docs/usage/getting-started/running.md

```diff
@@ -1,70 +1,88 @@
 
-If someone else is hosting Renovate for you, or you are using the Mend Renovate App on GitHub, then you can skip ahead to the [installing & onboarding](./installing-onboarding.md) page.
+If you're using the Mend Renovate App on GitHub, or if someone else is hosting Renovate for you, skip ahead to the [installing & onboarding](./installing-onboarding.md) page.
 
 ## Self-Hosting Renovate
 
-Self-hosting Renovate means that you are the "administrator" of the bot, which entails:
+When self-hosting Renovate you're the "administrator" of the bot, this means you:
 
-- You provide infrastructure for it to run on,
-- You provision its global config,
-- You ensure it's running regularly,
-- You ensure Renovate bot itself is updated
+- provide the infrastructure that Renovate runs on,
+- provision Renovate's global config,
+- make sure Renovate bot runs regularly,
+- make sure Renovate bot itself is updated
 
-Read our section on [Self-hosting on Windows](./installing-onboarding.md#self-hosting-on-windows) to prevent line endings from confusing Renovate bot.
+If you're self-hosting Renovate on Windows, read [Self-hosting on Windows](./installing-onboarding.md#self-hosting-on-windows) to prevent line endings from confusing Renovate bot.
 
 ### Available distributions
 
 #### npm package (CLI)
 
 Renovate's Open Source CLI is built and distributed as the npm package `renovate`.
-You can run this directly in any Node.js environment - even via `npx` - and it will process all the repositories it is configured with, before exiting.
-When you install Renovate from npm it naturally does not come bundled with any third-party tools or languages such as Ruby, Python, Composer, Bundler, Poetry, etc.
+You can run this package in any Node.js environment - even via `npx` - and it will process all the repositories it is configured with, before exiting.
+
+Renovate's npm package does not bundle any third-party tools or languages like Ruby, Python, Composer, Bundler, Poetry, etc.
 If you need Renovate to support any non-npm lock files like Bundler then you'll need to make sure all required third-party tools are pre-installed in the same environment alongside Renovate before you run it.
 
 The `renovate` npm package is compatible with all of Renovate's supported platforms.
 
-Renovate requires Node.js `>=14.15.0` and Git `>=2.33.0`.
+Renovate requires Node.js `>=18.12.0` and Git `>=2.33.0`.
+
+#### Docker images
+
+Renovate is also distributed as Docker images on Docker Hub (`renovate/renovate`) and GitHub container registry (`ghcr.io/renovatebot/renovate`).
+These Docker images work on all the hosting platforms that Renovate supports.
+
+You must run the Docker images in a `linux/amd64` container.
+We're working on `linux/arm64` container support.
+You can't run the Docker images on a Windows or macOS container.
+
+In general, you can run Renovate on Windows with global mode.
```

Comment:
First we say:

> You can't run the Docker images on a Windows or macOS container.

And then:

> In general, you can run Renovate on Windows with global mode.

I think we need more explanation for the "run Renovate on Windows with global mode" part. For example:

- Explain what global mode does, or link to the relevant section of the docs
- Be clear what version of Renovate you can run on Windows
- Any other thing that I'm forgetting, or don't know about 😄

Reply:
you can run renovate native on windows and macos via npm package. then you need to install required tool manually.

i don't think that there are macos containers. but windows container which run windows inside docker container.

you can run renovate docker images in docker desktop. on windows you need to use wsl. macos will use the `linux/amd64` even on modern arm mac's (changes soon with v36).

Reply:
Thanks for the explanation. ❤️ 

But I still don't know what we want to put in the docs based on your explanation. Can you write a suggestion for me?